README.md

```diff
@@ -1144,6 +1146,9 @@ res, err := descopeClient.Management.Project().Clone("new-project-name", "")
 if err == nil {
 		fmt.Println(cloneRes)
 }
+
+// Delete the current project
```

Comment:
cool, noting that because we use current project, I would add a note here that reminder user that following request may fail

something like

```suggestion
// Delete the current project. Kindly note that following calls on the `descopeClient` are most likely to fail because the current project has been deleted 
```